app/main.py

```diff
@@ -1,3 +1,56 @@
+import math
+
+
 class Vector:
-    # write your code here
-    pass
+    def __init__(self, x, y):
+        self.x = round(x, 2)
+        self.y = round(y, 2)
+
+    def __add__(self, other):
+        if isinstance(other, Vector):
+            return Vector(self.x + other.x, self.y + other.y)
+        else:
+            raise TypeError
+
```

Comment:
For every errors need to add a message about this error. For example: TypeError(f"{other} is not {type(Vector)}") It's good practice.